### PR TITLE
chore: add pull request template

### PR DESCRIPTION
**Pull request by QuakeEye**
## Description

This pull request adds a pull request template to ensure everything is done for each pull request that is required by our own standards.

## Testability

You can not test this, until creating a new pull request.

## Checklist
- [x] Set the proper pull request name
- [x] Merged main into your branch

### CODE REVIEW

File: .github/pull_request_template.md
Comment (joachimdekker) [inline .github/pull_request_template.md:1]: Kan je de headers niet beter maken van:

# Description
Propose a new change here

# Testability
How to test it...

Ik vind dat beschrijvender, want als ik het nu lees is het voor iemand die de PR gaat lezen nogal gek.

**Review by joachimdekker — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md
Comment (QuakeEye) [reply .github/pull_request_template.md:1]: Something like this?

## Description

*Desribe what you have done in this pull request*

## Testability

*Describe how to test your changes*

## Checklist
- [ ] Set the proper pull request name
- [ ] Merged main into your branch

**Review by joachimdekker — APPROVED** [review]
LGTM

File: Aplib.Core/Belief/IBeliefSet.cs [hunk 1]
```diff
@@ -5,9 +5,9 @@
...
     public interface IBeliefSet
     {
-      /// <summary>
-      /// Updates all beliefs in the belief set.
-      /// </summary>
-      void UpdateBeliefs();
+        /// <summary>
+        /// Updates all beliefs in the belief set.
+        /// </summary>
+        void UpdateBeliefs();
```
Comment (JensSteenmetz) [inline Aplib.Core/Belief/IBeliefSet.cs:11]: Undo these changes, as they are already made in 'feat: add MemoryBelief' and they don't really fit in this PR

**Review by JensSteenmetz — CHANGES_REQUESTED** [review]

**Review by JensSteenmetz — APPROVED** [review]

**Pull request merged**